Core/Core/Logging/SpeckleLog.cs

```diff
@@ -264,9 +264,18 @@ private static string GetUserIdFromDefaultAccount()
     return id;
   }
 
-  private static FileVersionInfo GetFileVersionInfo()
+  private static FileVersionInfo? GetFileVersionInfo()
   {
-    var assembly = Assembly.GetExecutingAssembly().Location;
+    // For some reason, Location will return null inside GoDot runtimes,
+    // Despite `Location` being decorated as non-nullable
+    string? assembly = Assembly.GetExecutingAssembly().Location;
+
+    // ReSharper disable once ConditionIsAlwaysTrueOrFalseAccordingToNullableAPIContract
+    if (assembly == null)
```

Comment:
Hi @JR-Morgan 

I tracked down the issue, the problem is the assembly isn't null its empty.
Using a Null Or Empty check correctly returns a null value.
if (string.IsNullOrEmpty(assembly))

Reply:
ah yep ok, that makes more sense now. I've updated it to do a `IsNullOrEmpty`

Thanks a lot!